src/chef.js

```diff
@@ -15,6 +15,35 @@ function main() {
   controller.on(['bot_channel_join', 'direct_mention'], (bot, event) => {
   	bot.reply(event, 'hey gang')
   })
+  
+  controller.hears(['if i say'], (bot, message) => {
+    const msgText = message.text;
+
+    if (!msgText.includes('you say')) {
+      return bot.reply(message, 'can u not ☹️')
+    }
+
+    const trigger = msgText.split('if i say')[1].split('you say')[0].trim()
+    const response = msgText.split('if i say')[1].split('you say')[1].trim()
```

Comment:
can use array destructuring to replace those 2 lines with:
`const [trigger, response] = msgText.split('if i say')[1].split('you say').map(s=>s.trim());`